options/options.js

```diff
@@ -230,137 +287,160 @@ function loginBitbucket(param) {
+      headers: headers,
+      method: 'POST',
+      dataType: 'json',
+      contentType: 'application/x-www-form-urlencoded',
+      data: {
+        grant_type: 'password',
+        username: username,
+        password: password
+      }
+    })
+    .done(response => {
+      return $.getJSON(
+          `${baseUrl}/user`, {
+            access_token: response.access_token
+          }
+        )
+        .done(user => {
+          chrome.storage.sync.set({
+            scm: param.scm,
+            user: user.username,
+            token: response.refresh_token,
+            baseUrl: baseUrl
+          }, () => {
+            location.reload();
+          });
+          chrome.storage.local.get('tab', (item) => {
+            if (item.tab) {
+              chrome.tabs.reload(item.tab);
+            }
+          });
+        });
+    })
+    .fail(() => {
+      $('.error').show();
+    })
 }
 
 function loginGitLabOauth(param) {
   const username = param.username;
   const password = param.password;
   const baseUrl = param.baseUrl;
   const headers = {}
-  const domain = baseUrl.replace('/api/v4','');
+  const domain = baseUrl.replace('/api/v4', '');
   $.ajax({
-    url: `${domain}/oauth/token`,
-    headers: headers,
-    method: 'POST',
-    dataType: 'json',
-    crossDomain: true,
-    contentType: 'application/x-www-form-urlencoded',
-    data: {
-      grant_type: 'password',
-      username: username,
-      password: password
-    }
-  })
-  .done(response => {
-    return $.getJSON(
-      `${baseUrl}/user`,
-      { access_token: response.access_token }
-    )
-    .done(user => {
-      chrome.storage.sync.set({scm: param.scm, user: user.username, token: {type : 'oAuth', token :response.access_token}, baseUrl: baseUrl}, () => {
-        location.reload();
-      });
-      chrome.storage.local.get('tab', (item) => {
-        if(item.tab) {
-          chrome.tabs.reload(item.tab);
-        }
-      });
-    });
-  })
-  .fail(err => {
+      url: `${domain}/oauth/token`,
+      headers: headers,
+      method: 'POST',
+      dataType: 'json',
+      crossDomain: true,
+      contentType: 'application/x-www-form-urlencoded',
+      data: {
+        grant_type: 'password',
+        username: username,
+        password: password
+      }
+    })
+    .done(response => {
+      return $.getJSON(
+          `${baseUrl}/user`, {
+            access_token: response.access_token
+          }
+        )
+        .done(user => {
+          chrome.storage.sync.set({
+            scm: param.scm,
+            user: user.username,
+            token: {
+              type: 'oAuth',
+              token: response.access_token
+            },
+            baseUrl: baseUrl
+          }, () => {
+            location.reload();
+          });
+          chrome.storage.local.get('tab', (item) => {
+            if (item.tab) {
+              chrome.tabs.reload(item.tab);
+            }
+          });
+        });
+    })
+    .fail(err => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/083dcd77ddb6869d2b7dc2cdf2f7452c410d4901/options/options.js#L374"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>